packages/trpc/server/routers/viewer/workflows/util.ts

```diff
@@ -781,6 +780,63 @@ export const getEventTypeWorkflows = async (
   userId: number,
   eventTypeId: number
 ): Promise<z.infer<typeof ZWorkflows>> => {
-  const rawEventType = await EventTypeRepository.findById({ id: eventTypeId, userId });
-  return rawEventType?.workflows;
+  const workflows = await prisma.workflow.findMany({
```

Comment:
@CarinaWolli i tested this and im pretty sure this works as expected - this query has been changed a bit from the one in eventTypes repo. 

Targeting the workflow table directly instead of the event types. This generates some nicer SQL for what we needed. Also doesnt over fetch the data outside of `eventTypes.workflows` that was un-needed in this call

Reply:
I think this would be a good opportunity to add as a method in the workflow repository.

Reply:
I have a separate issue for this! Gonna be extracting a lot of the current workflow repository to a service and handling that migration there. Mind if we pick that up there?

Reply:
Excited to hear! I'm ok with that.